src/bridge.js

```diff
@@ -15,15 +18,15 @@ module.exports.bridge = function(port, config) {
             onEvent: function(request, context) {
                 // events from matrix
                 const event = request.getData();
-                console.log(event)
+                log.info(event);
```

Comment:
This might be quite verbose for `info` level, perhaps make it `debug`? Also might want a string before it to explain what you are logging here.